README.md

```diff
@@ -25,26 +25,45 @@ To build, run `mvn clean install`.
 | master  | 0.6.0-SNAPSHOT | 0.11.0-SNAPSHOT         |
 
 
-## Example
 
-First add the required jars to your Apache Brooklyn release (see "Future Work" for discussion 
-of OSGi): 
+## Installation AMP
 
-    BROOKLYN_HOME=~/repos/apache/brooklyn/brooklyn-dist/dist/target/brooklyn-dist/brooklyn/
-    BROOKLYN_AWS_ELB_REPO=~/repos/cloudsoft/brooklyn-aws-elb
-    MAVEN_REPO=~/.m2/repository
+Start AMP and add the feature
+    
+    # Start Brooklyn/AMP karaf
+    ${BROOKLYN_HOME}/bin/karaf
+    
+    # Add io.cloudsoft.aws.elb feature repo
+    feature:repo-add mvn:io.cloudsoft.aws.elb/feature/0.6.0-SNAPSHOT/xml/features
+    
+    # Add the feature
+    feature:install amp-aws-elb
 
-    AWS_SDK_VERSION=1.10.53
-    BROOKLYN_AWS_ELB_VERSION=0.5.0-SNAPSHOT
+## Installation Brooklyn
+
+Start Brooklyn and add the feature
+        
+    # Update the setenv file
```

Comment:
I think it would be clearer to include these instructions in a separate code block. e.g.:
```
Append the following to `$BROOKLYN_HOME/bin/setenv`:

    # Add io.cloudsoft domain to ClassLoaderUtils whitelist so that io.cloudsoft bundles are also scanned for classes.
    WHITELIST='org.apache.brooklyn.*|io.brooklyn.*|io.cloudsoft.*'
    export EXTRA_JAVA_OPTS="-Dorg.apache.brooklyn.classloader.fallback.bundles=${WHITELIST} ${EXTRA_JAVA_OPTS}"

Then start Brooklyn and install the feature:

    ${BROOKLYN_HOME}/bin/karaf
    
    # Add io.cloudsoft.aws.elb feature repo
    feature:repo-add mvn:io.cloudsoft.aws.elb/feature/0.6.0-SNAPSHOT/xml/features
    
    # Add the feature
    feature:install amp-aws-elb
```